src/YawPitchControl/utils.js

```diff
@@ -6,3 +8,42 @@ export function toAxis(source, offset) {
 		return acc;
 	}, {});
 }
+
+/**
+ * Returns a number value indiciating the version of Chrome being used,
+ * or otherwise `null` if not on Chrome.
+ *
+ * Ref: https://github.com/immersive-web/cardboard-vr-display/pull/19
+ */
+export const getChromeVersion = (() => {
+	const match = userAgent.match(/.*Chrome\/([0-9]+)/);
+	const value = match ? parseInt(match[1], 10) : -1;
+
+	return () => value;
+})();
+
+/**
+ * In Chrome m65, `devicemotion` events are broken but subsequently fixed
+ * in 65.0.3325.148. Since many browsers use Chromium, ensure that
+ * we scope this detection by branch and build numbers to provide
+ * a proper fallback.
+ * https://github.com/immersive-web/webvr-polyfill/issues/307
+ */
+export const isChromeWithoutDeviceMotion = (() => {
+	let value = false;
+
+	if (getChromeVersion() === 65) {
+		const match = userAgent.match(/.*Chrome\/([0-9.]*)/);
```

Comment:
@jongmoon

I don't think userAgent is a lower case, do I?

If it has, add i flag. `/Chrome/i`

Reply:
I agree!